test/ui-test/src/screens/components/saved_address_popup.py

```diff
@@ -52,15 +56,16 @@ def add_saved_address(self, name: str, address: str):
         self._name_text_edit.text = name
         self._address_text_edit.clear(verify=False)
         self._address_text_edit.type_text(address)
-        self.verify_network_selector_enabled()
-        self._add_networks_selector.click(1, 1)
-        self.set_ethereum_mainnet_network(True)
-        self.set_optimism_mainnet_network(True)
-        self.set_arbitrum_mainnet_network(True)
-        self._save_add_address_button.click() # i click it twice to close the network selector pop up
-        self.verify_ethereum_mainnet_network_tag_present()
-        self.verify_otimism_mainnet_network_tag_present()
-        self.verify_arbitrum_mainnet_network_tag_present()
+        if address.startswith("0x"):
+            assert self.verify_network_selector_enabled(), f'Network selector is not active'
+            self._add_networks_selector.click(1, 1)
+            self.set_ethereum_mainnet_network(True)
+            self.set_optimism_mainnet_network(True)
+            self.set_arbitrum_mainnet_network(True)
+            self._save_add_address_button.click() # i click it twice to close the network selector pop up
+            assert self.verify_ethereum_mainnet_network_tag_present(), f'Ethereum Mainnet network tag is not present'
```

Comment:
oh, got it

Reply:
well, verify method should verify something, rename it or move the assertation into the method

Reply:
ya thats true, i wanted to have 1 method that can return bool so i can further call it and assert or assert NOT (instead of keeping 2 separate methods), i need to check that element is present and is not present. i will rename @StateOf-Vlado

Reply:
verify_ethereum_mainnet_network_tag_present can be is_ethereum_mainnet_network_tag_present - in that case, it will be clear that method return something, for now, according to the name I expect that it verify somthing

Reply:
i agree, it is confusing, will rename!